ledger/metrics/src/test/lib/scala/com/daml/telemetry/TelemetrySpecBase.scala

```diff
@@ -12,7 +12,7 @@ import scala.jdk.CollectionConverters._
 
 trait TelemetrySpecBase {
 
-  protected val anInstrumentationName = "com.daml.telemetry.TelemetrySpec"
+  protected val anInstrumentationName: String = classOf[TelemetrySpecBase].getCanonicalName
```

Comment:
Wouldn't it more readable if it used the actual runtime class name (i.e., of `this`)?

Reply:
Makes sense, thanks.